compiler/luci-interpreter/src/kernels/MirrorPad.cpp

```diff
@@ -1,5 +1,6 @@
 /*
  * Copyright (c) 2021 Samsung Electronics Co., Ltd. All Rights Reserved
+ * Copyright 2019 The TensorFlow Authors. All Rights Reserved.
```

Comment:
@binarman PTAL. Is it enough as attribution for Apache 2.0 code?

Reply:
Yes, it should be ok

Reply:
Could you point where this code is located in TF sources?

I can not find it, so I am not sure we need this line.

Reply:
It's in [``tensorflow/lite/kernels/internal/reference/pad.h``](https://github.com/tensorflow/tensorflow/blob/919f693420e35d00c8d0a42100837ae3718f7927/tensorflow/lite/kernels/internal/reference/pad.h#L38), modified in extended (it supported up to 5d, I've limited support to 4).